addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -47,6 +47,13 @@ class AccountInvoice(models.Model):
 
     l10n_it_einvoice_id = fields.Many2one('ir.attachment', string="Electronic invoice", copy=False)
 
+    @api.onchange('partner_id', 'company_id')
+    def _onchange_partner_id(self):
+        res = super()._onchange_partner_id()
+        if self.company_id.country_id.code != 'IT':
```

Comment:
What is this?  It depends on partner_id, but the partner is not in the logic....

Reply:
@jva-odoo @mgh-odoo

Reply:
hello @jco-odoo,
It has been overridden, so we kept field dependencies as per it's parent definition. But actually it is quite old PR so it is still having account.invoice model, I will rebase it with saas-12.5 and adapt changes and I think we will now need to override _onchange_journal since it is refactored quite a bit. I'll ping you once I rebased it.

Reply:
@mgh-odoo Ok to rebase it, but this should be done in v12 as it is a fix.  Also doing this with an onchange is not a good idea as you add a state for it and by default it is in 'to_send'.  Why can not we just check if the company is Italian?

Reply:
@jco-odoo  are you sure? Because, in the task, it is suggested to fix this in master considering it is a non-blocking issue.

Reply:
@jco-odoo non-blocking, but a bug nevertheless..I'll be fixing it for v12 anyways :+1: